build/debs/kubelet-postinst.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -x /bin/systemctl ] && systemctl is-active systemd-resolved > /dev/null 2>&1 ; then
```

Comment:
How about `if [[ -x /bin/systemctl && $(systemctl is-active systemd-resolved) == "active" ]]; then`

Reply:
Does parsing the output provide any value above the error code? I think we should avoid parsing output if possible, otherwise we will have a potential future bug if the output format changes.

Reply:
Either way wfm

Reply:
`systemctl is-active systemd-resolved --quiet` turns out to be the ideal form, as per https://www.freedesktop.org/software/systemd/man/systemctl.html#is-active%20PATTERN%E2%80%A6